gbpservice/nfp/config/mode_shift.conf

```diff
@@ -0,0 +1,24 @@
+#------------------------#
+# MODE SHIFT INSTRUCTION #
+#------------------------#
+# NOTE: The present scope of the script supports only ADVANCED to ENTERPRISE migration.
+#       Please see that the machine has atleast 16 GB RAM left.
+
+FROM=advanced
+TO=enterprise
```

Comment:
This was just added for future extension. If we don't have any requirement of other combinations of FROM and TO, we will remove this.